components/Common/Tabs/index.tsx

```diff
@@ -16,7 +16,10 @@ const Tabs: FC<PropsWithChildren<TabsProps>> = ({
   children,
   ...props
 }) => (
-  <TabsPrimitive.Root {...props}>
+  <TabsPrimitive.Root
+    {...props}
+    className={[styles.tabsRoot, props.className].filter(Boolean).join(' ')}
```

Comment:
```suggestion
  <TabsPrimitive.Root
    {...props}
    className={classNames(styles.tabsRoot, className)}
```

You can use the `classNames` function that we use throughout the project to combine CSS classes. Also, it may be better to add `className` type to `TabsProps` and use it instead of using `props.className`

Reply:
Is there reason to add `className` as an explicit prop, rather than leaving the default type definition from it from the underlying `div` props in the primitive?

Reply:
There are over 240 props that can be used in this component, IMO seeing only the places that concern us (where we modify) makes the code more readable